cfme/configure/configuration/server_settings.py

```diff
@@ -451,15 +451,280 @@ def _save_action(self, view, updates, reset):
+            ex. auth_settings.update_form({"hours_timeout": hours, "mode": "Amazon"}, reset=True)
+        """
+        view = navigate_to(self, self.auth_mode)
+        view.fill(updates)
+        try:
+            view.validate.click()
+            view.flash.assert_message(
+                '{} Settings validation was successful'.format(
+                    view.authentication_mode.selected_option))
+        except AttributeError:
+            logger.info("View doesn't have validate button")
+        if reset:
+            view.reset_button.click()
+            view.flash.assert_message('All changes have been reset')
+        else:
+            view.save_button.click()
+            flash_message = (
+                'Authentication settings saved for {} Server "{} [{}]" in Zone "{}"'.format(
+                    self.appliance.product_name,
+                    self.appliance.server_name(),
+                    self.appliance.server_id(),
+                    self.appliance.server.zone.name))
+            view.flash.assert_message(flash_message)
+
+    @property
+    def auth_settings(self):
+        """ Authentication view fields values """
+        view = navigate_to(self, self.auth_mode)
+        return view.read()
+
+    def set_auth_mode(self, reset=False, **kwargs):
+        """ Set up authentication mode
+
+        Args:
+            reset: Set True, to reset all changes for the page. Default value: False
+
+        kwargs: A dict of keyword arguments used to initialize auth mode
+                if you want not to use yamls settings,
+                mode='your_mode_type_here' key/value should be a mandatory in your kwargs
+            ex. auth_settings.set_auth_mode(
+            reset= True, mode='Amazon', access_key=key, secret_key=secret_key)
+        """
+        form_to_fill = {}
+        if kwargs:
+            self.auth_mode = kwargs['mode'].capitalize()
+            for key, value in kwargs.items():
+                if key not in ['mode', 'default_groups']:
+                    if key == 'hosts':
+                        assert len(value) <= 3, "You can specify only 3 LDAP hosts"
+                        for enum, host in enumerate(value):
+                            form_to_fill["ldap_host_{}".format(enum + 1)] = host
+                    elif key == 'user_type':
+                        form_to_fill[key] = self.user_type_dict[value]
+                    else:
+                        form_to_fill[key] = value
+        else:
+            self.auth_mode = 'Database'
+        self._update_form(form_to_fill, reset)
+
+
+@navigator.register(AuthenticationSetting, 'Details')
+class DetailsAuth(CFMENavigateStep):
+    VIEW = ServerAuthenticationView
+    prerequisite = NavigateToAttribute('appliance.server', 'Details')
+
+    def step(self):
+        self.prerequisite_view.authentication.select()
+
+
+@navigator.register(AuthenticationSetting)
```

Comment:
**Required**

Add nav destination names here for consistency - 'Ldap' is specified below, but no 'Database' here or 'Ldaps'  on L701